.github/workflows/stubtest.yml

```diff
@@ -55,3 +56,23 @@ jobs:
         run: sudo apt install -y $(python tests/get_apt_packages.py)
       - name: Run stubtest
         run: python tests/stubtest_third_party.py --num-shards 4 --shard-index ${{ matrix.shard-index }}
+
+  # https://github.community/t/run-github-actions-job-only-if-previous-job-has-failed/174786/2
+  create-issue-on-failure:
+    name: Create an issue if stubtest failed
+    runs-on: ubuntu-latest
+    needs: [stubtest-stdlib, stubtest-third-party]
+    if: ${{ github.repository == 'python/typeshed' && always() && (needs.stubtest-stdlib.result=='failure' || needs.stubtest-third-party.result=='failure') }}
+    permissions:
```

Comment:
I can't find it in GitHub docs: does it override or join permissions?

Reply:
The important thing is that it works. It doesn't matter whether it has access to the code in the repo, because it only creates an issue.